sample/Sample/ContentPopup/ContentPopupMainPage.xaml.cs

```diff
@@ -29,7 +29,7 @@ public ContentPopupMainPage()
             BindingContext = new ContentPopupTestModel();
         }
 
-        async void ItemSelected(object sender, ItemTappedEventArgs args)
+        async void ItemTapped(object sender, ItemTappedEventArgs args)
```

Comment:
The behavior of `ItemTapped` event handler for test Pages are looks almost same. How about extracting a method or object such as Command for using?
`Commanding`  - https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/data-binding/commanding?view=net-maui-7.0

Reply:
Good review. but I will consider it another PR related to `command`.